rclcpp/include/rclcpp/logger.hpp

```diff
@@ -170,6 +170,15 @@ class Logger
   RCLCPP_PUBLIC
   void
   set_level(Level level);
+
+  /// Get effective level for current logger.
```

Comment:
Users might not know the whole meaning of effective level here, especially for a logger that has ancestors.
https://github.com/ros2/rcutils/blob/90461677df74ed2dc3c5a4802a156a16f9082820/include/rcutils/logging.h#L416-L421 ?

Reply:
that is true, should be added to doc section.

Reply:
addressed by https://github.com/ros2/rclcpp/pull/2141/commits/b185c701093ffbe3d711c61fe07794e7875bc6f2